CITATION.cff

```diff
@@ -0,0 +1,243 @@
+  - given-names: Travis
+    family-names: Jensen
+  - given-names: Bryce
+    family-names: Allen
+    orcid: 'https://orcid.org/0000-0002-0804-8127'
+    affiliation: Differentiated Therapeutics
+  - given-names: Dominik
+    family-names: Wille
+    affiliation: Freie Universität Berlin
+  - given-names: Victoria
+    family-names: Lim
+    orcid: 'https://orcid.org/0000-0003-4030-9312'
+    affiliation: University of California Irvine
+  - given-names: Hyungro
+    family-names: Lee
+    orcid: 'https://orcid.org/0000-0002-4221-7094'
+    affiliation: Pacific Northwest National Laboratory
+  - given-names: Mohammad Soroush
+    family-names: Barhaghi
+    orcid: 'https://orcid.org/0000-0001-8226-7347'
+    affiliation: Wayne State University
+  - given-names: Zhiyi
+    family-names: Wu
+    affiliation: Exscientia plc
+    orcid: 'https://orcid.org/0000-0002-7615-7851'
+    email: william@zhiyiwu.me
+  - given-names: Wei-Tse
+    family-names: Hsu
+    orcid: https://orcid.org/0000-0001-6167-5480
+    affiliation: University of Colorado Boulder    
+  - given-names: Alexander
+    family-names: Schlaich
+    orcid: 'https://orcid.org/0000-0002-4250-363X'
+    affiliation: University of Stuttgart
+  - given-names: Jérôme
+    family-names: Hénin
+    orcid: 'https://orcid.org/0000-0003-2540-4098'
+    affiliation: CNRS
+  - given-names: Thomas T.
+    family-names: Joseph
+    orcid: 'https://orcid.org/0000-0003-1323-3244'
+    affiliation: University of Pennsylvania
+  - given-names: Irfan
+    family-names: Alibay
+    orcid: 'https://orcid.org/0000-0001-5787-9130'
+    affiliation: Open Molecular Software Foundation
+  - given-names: Pascal T.
+    family-names: Merz
+    affiliation: PM Scientific Consulting
+    orcid: 'https://orcid.org/0000-0002-7045-8725'
+  - given-names: Domenico
+    family-names: Marson
+    orcid: 'https://orcid.org/0000-0003-1839-9868'
+    affiliation: University of Trieste
+  - given-names: Shujie
+    family-names: Fan
+    orcid: 'https://orcid.org/0000-0002-0271-2760'
+    affiliation: Arizona State University
+  - given-names: Jan
+    family-names: Janssen
+    affiliation: Max-Planck-Institut for Sustainable Materials
+    orcid: 'https://orcid.org/0000-0001-9948-7119'
+  - given-names: Haoxi
+    family-names: Li
+    orcid: 'https://orcid.org/0009-0004-8369-1042'
+    affiliation: University of North Carolina
+  - given-names: Helmut
+    family-names: Carter
+    orcid: https://orcid.org/0000-0003-0273-4107    
+    affiliation: City University of New York
+  - given-names: David
+    family-names: Mobley
+    orcid: 'https://orcid.org/0000-0002-1083-5533'
+    affiliation: University of California Irvine    
+  - given-names: Michael R.
+    family-names: Shirts
+    orcid: 'https://orcid.org/0000-0003-3249-1097'
+    affiliation: University of Colorado Boulder
+  - given-names: Jennifer A.
+    family-names: Clark
+    affiliation: National Institute of Standards and Technology
```

Comment:
@jaclark5 is this information correct (name, affiliation)

Do you have an ORCID that we could add?

Reply:
Yes it's correct. https://orcid.org/0000-0003-4897-5651

Reply:
thanks, updated